src/groups/product_group.jl

```diff
@@ -344,7 +344,7 @@ function Base.show(
 end
 
 function get_coordinates_lie!(
-    Pr𝔤::LieAlgebra{𝔽,Op,LieGroup{𝔽,Op,M}}, c, X, B::DefaultLieAlgebraOrthogonalBasis
+    Pr𝔤::LieAlgebra{𝔽,Op,LieGroup{𝔽,Op,M}}, c, X
```

Comment:
Why isn't ` B` needed here?

Reply:
`get_coordinates_lie` is already the specific (Layer3) function for the `DefaultLieAlgebraOrthogonalBasis`, so this function does not need a basis any longer.

Reply:
There are variants of `DefaultLieAlgebraOrthogonalBasis` with real and complex coefficients and this information needs to be propagated here somehow (and I don't like passing just the number system, it's too much hassle).

Reply:
Anyway, `B` is even used inside so this function can't work without it.

Reply:
Yes, and IMO it was a bad idea to not pass the basis to layer-3 functions. I think this is a good time to stop doing it. I'm planning to redo ManifoldsBase dispatch accordingly when I have some spare time.

Reply:
I did not have a strong opinion on that when it was introduced but I do have a strong opinion now after working with the current design a bit.

Reply:
Sure, sometimes it needs some time and using stuff to see what is better.

Reply:
But yeah, I merged master with the `ValidationLieGroup` but will add `B` than back in tomorrow and check that this is done everywhere for `get_vector_lie` and `get_coordinates_lie` then.